frontend/src/app/libs/event-info/services/event-info.service.ts

```diff
@@ -54,6 +65,23 @@ export class EventInfoService {
 					label: 'Some label32',
 				},
 			]),
+			solution: {
+				status: SolutionStatus.Draft,
+				value: 'Write your answer ...'
+			}
+		},
+		{
+			id: 'key4',
+			taskName: 'task 4',
+			taskType: TasksTypes.Text,
+			mentorCheck: false,
+			maxScore: 60,
+			text: '',
+			question: '<p>Text question ...</p><p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Itaque magnam molestias qui assumenda quia animi iste odio ipsam debitis, corrupti dolorum id dicta sapiente eos! Minima porro ex iure omnis?</p>',
+			solution: {
```

Comment:
i think for this type of task we can skip solution section since it will be validated by mentor.